pkg/cmd/server/api/v1/types.go

```diff
@@ -881,3 +888,12 @@ type LDAPQuery struct {
 	// Filter is a valid LDAP search filter that retrieves all relevant entries from the LDAP server with the base DN
 	Filter string `json:"filter"`
 }
+
+type AdmissionConfig struct {
+	// PluginConfig allows specifying a configuration file per admission control plugin
+	PluginConfig map[string]string `json:"pluginConfig"`
```

Comment:
> We have AlwaysPullImages upstream that I wrote. It doesn't currently take any config, but I suppose it could. I'm planning on writing AlwaysPullBuildImages in Origin specific to builds, or we could create something like AlwaysPullAllImages that maybe embeds AlwaysPullImages and adds the build specific logic. What do you all think is the best path forward?

How can you tell one kind of image from another?  Label selector on a pod?  If so, that would suggest a single admission plugin with a configurable set of Label selectors.